docs/pages/docs/Components/Inputs/switch.en-US.mdx

```diff
@@ -42,11 +42,19 @@ render(<SimpleSwitch />)`} noInline />
 
 ### `colorScheme`
 
-The background color of switch when is active.
+The background to-tint color of switch when is active.
```

Comment:
To match with other `colorScheme` prop description: `The colorScheme property, will define background color of switch when is active`